.jenkins

```diff
@@ -5,7 +5,6 @@ pipeline {
   agent any
 
   environment {
-    PROJ_PATH = '/go/src/github.com/vapor-ware/synse-amt-plugin'
     IMAGE_NAME = 'vaporio/amt-plugin'
     GOVERSION = '1.12'
```

Comment:
Did you want to normalize this based on the env var in the container?

Reply:
yup! that'll come in the next PR with other general CI updates. (: